mesonbuild/compilers.py

```diff
@@ -59,12 +59,15 @@ def is_library(fname):
                           # See https://github.com/mesonbuild/meson/pull/509
                           'debug' : ['-O0', '-g'],
                           'debugoptimized' : ['-O2', '-g'],
-                          'release' : ['-O3']}
+                          'release' : ['-O3'],
+                          'minsize' : ['-Os', '-g']}
 
 msvc_buildtype_args = {'plain' : [],
                        'debug' : ["/MDd", "/ZI", "/Ob0", "/Od", "/RTC1"],
                        'debugoptimized' : ["/MD", "/Zi", "/O2", "/Ob1"],
-                       'release' : ["/MD", "/O2", "/Ob2"]}
+                       'release' : ["/MD", "/O2", "/Ob2"],
+                       'debugoptimized' : ["/MD", "/Zi", "/Os", "/Ob1"],
```

Comment:
Shouldn't this be `minsize`? Also it should be a variant of `release`, so `['/MD', '/Os', '/Obs']`.